app/models/game.rb

```diff
@@ -1,7 +1,17 @@
 class Game < ActiveRecord::Base
+  attr_accessor :creator_plays_as_black # for checkbox access
   has_many :pieces, dependent: :destroy
-  has_many :users
```

Comment:
split the has_many relationship into 2 many-to-one relationships so we can implement foreign keys later